output/Form.md

```diff
@@ -40,7 +40,40 @@ title: Yerun Open Science Award form
 
 -   *Can the practice be reused in other departments or institutions?*
 
-<!--# blank field, max 400 words -->
+The idea is to record all the steps that are taken so that they are accessible
+and reusable by everyone, regardless of their level of experience with the
```

Comment:
I don't think this is exactly true. IMO using git and navigating the history of a project requieres a relatively high level of expertise. The same can be said of Opal and R/Rmarkdown